runtime/missingmember_test.go

```diff
@@ -1619,20 +1619,12 @@ pub contract ItemNFT: NonFungibleToken {
 
        // get a reference to the garment that item stores
        pub fun borrowGarment(): &GarmentNFT.NFT? {
-           let garmentOptional <- self.garment <- nil
-           let garment <- garmentOptional!
-           let garmentRef = &garment as auth &GarmentNFT.NFT
-           self.garment <-! garment
-           return garmentRef
+           return &self.garment as auth &GarmentNFT.NFT?
```

Comment:
Does not have to be `auth`
```suggestion
           return &self.garment as &GarmentNFT.NFT?
```